uc-client/examples/uc-cli.rs

```diff
@@ -0,0 +1,278 @@
+                }
+            }
+        }
+
+        Commands::Commits {
+            name,
+            start_version,
+            end_version,
+        } => {
+            println!("Resolving table: {}", name);
+
+            // First, get the table metadata to obtain table_id and storage_location
+            let table = match client.get_table(&name).await {
+                Ok(table) => {
+                    println!(
+                        "✓ Table resolved: {} (ID: {})",
+                        table.full_name(),
+                        table.table_id
+                    );
+                    table
+                }
+                Err(e) => {
+                    eprintln!("✗ Failed to resolve table: {}", e);
+                    std::process::exit(1);
+                }
+            };
+
+            println!("Fetching commits for table...");
+
+            let mut request = CommitsRequest::new(&table.table_id, &table.storage_location);
+
+            // Set start_version - default to 0 if not provided
+            let start = start_version.unwrap_or(0);
+            request = request.with_start_version(start);
+
+            if let Some(end) = end_version {
+                request = request.with_end_version(end);
+            }
+
+            match client.get_commits(request).await {
+                Ok(response) => {
+                    println!("\n✓ Commits retrieved successfully\n");
+                    println!("Table:           {}", table.full_name());
+                    println!("Latest Version:  {}", response.latest_table_version);
+
+                    if let Some(commits) = response.commits {
+                        println!("\nCommits ({} total):", commits.len());
+                        println!(
+                            "{:<8} {:<20} {:<40} {:<15}",
+                            "Version", "Timestamp", "File Name", "Size"
+                        );
+                        println!("{}", "-".repeat(85));
+
+                        for commit in commits {
+                            let timestamp_str = commit
+                                .timestamp_as_datetime()
+                                .map(|dt| dt.format("%Y-%m-%d %H:%M:%S").to_string())
+                                .unwrap_or_else(|| {
+                                    format!("Invalid timestamp: {}", commit.timestamp)
+                                });
+
+                            println!(
+                                "{:<8} {:<20} {:<40} {:<15}",
+                                commit.version,
+                                timestamp_str,
+                                if commit.file_name.len() > 40 {
+                                    format!("{}...", &commit.file_name[..37])
+                                } else {
+                                    commit.file_name.clone()
+                                },
+                                format_bytes(commit.file_size)
+                            );
```

Comment:
This also seems like something that can exist in the `fmt::Display` for Commit.

Reply:
actually did standalone helper for now since this takes an opinion on formatting since we are printing it like a table (can optimize in future)